eng/pipelines/runtime.yml

```diff
@@ -288,11 +288,9 @@ jobs:
     buildConfig: Release
     runtimeFlavor: mono
     platforms:
-    - Android_x64
     - Android_arm
     - tvOS_arm64
     - iOS_arm
-    - iOS_x64
```

Comment:
This is probably not intentional right?

Reply:
Pretty much every change in runtime.yml is a temporary hack